server/index.js

```diff
@@ -8,7 +8,7 @@ import morgan from 'morgan';
 import dotenv from "dotenv";
 import logger from '../logger.js';
 import unknownEndpointHandler from './middleware/unknown-endpoint-handler.js';
-import expressErrorHandler from './middleware/express-error-handler.js';
+// import expressErrorHandler from './middleware/express-error-handler.js';
```

Comment:
A number of lines below are commented out as well, and presumably can be removed.